Unable to use frugally-deep with FetchContent

Hi @Dobiasd! First of all, thank you very much for developing and maintaining `frugally-deep`!

I would like to use `frugally-deep` with CMake `FetchContent`, however, I'm having some trouble.

I wrote this very simple `main.cpp` file
```cpp
// main.cpp
#include <fdeep/fdeep.hpp>

int main()
{
    return 0;
}
```
and the associated `CMakeLists.txt` file
```cmake
cmake_minimum_required(VERSION 3.16)
project(test_keras_model)

include(FetchContent)
FetchContent_Declare(
    functional_plus
    GIT_REPOSITORY https://github.com/Dobiasd/FunctionalPlus.git
    GIT_TAG master
    )

  FetchContent_Declare(
    frugally_deep
    GIT_REPOSITORY https://github.com/Dobiasd/frugally-deep.git
    GIT_TAG master
)

FetchContent_MakeAvailable(functional_plus frugally_deep)

add_executable(test_keras_model main.cpp)
target_link_libraries(test_keras_model frugally-deep::fdeep)
```
when I call `cmake .` I got the following error
```
-- ===( frugally-deep 0.15.19 )===
-- Configuring done
CMake Error at CMakeLists.txt:20 (add_executable):
  Target "test_keras_model" links to target "frugally-deep::fdeep" but the
  target was not found.  Perhaps a find_package() call is missing for an
  IMPORTED target, or an ALIAS target is missing?
```
there seems that `FetchContent_MakeAvailable()` is not able to generate the target `frugally-deep::fdeep`


cc @isorrentino

--- a/CMakeLists.txt
+++ b/CMakeLists.txt
@@ -19,6 +19,7 @@ if(NOT FDEEP_USE_TOOLCHAIN)
 endif()
 
 add_library(fdeep INTERFACE)
+add_library(${PROJECT_NAME}::fdeep ALIAS fdeep)
 target_include_directories(fdeep INTERFACE
   $<BUILD_INTERFACE:${CMAKE_CURRENT_LIST_DIR}/include>
   )

